community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/pipes/LegacySortPipe.scala

```diff
@@ -37,8 +37,13 @@ case class LegacySortPipe(source: Pipe, sortDescription: List[SortItem])
       sortWith((a, b) => compareBy(a, b, sortDescription)(state)).iterator
   }
 
-  def planDescription =
-    source.planDescription.andThen(this.id, "Sort", variables, sortDescription.map(item => LegacyExpression(item.expression)):_*)
+  def planDescription = {
+    val sorting = sortDescription.zipWithIndex.map {
+      case (sortItem, idx) => s"o$idx" -> sortItem.expression
```

Comment:
In my head was the word "ordinal" when I picked `o`. Not super important - the numbers are more important, since they decide what the sort order will be.